contracts/core/src/examples/Funder.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.24;
+
+import { XApp } from "src/pkg/XApp.sol";
+import { XGasPump } from "src/pkg/XGasPump.sol";
+import { ConfLevel } from "src/libraries/ConfLevel.sol";
+
+/**
+ * @title Funder
+ * @notice Example contract that shows how to use XGasPump
+ */
+contract Funder is XApp, XGasPump {
```

Comment:
Going to remove this example before mering

Reply:
why? examples are good!

Reply:
guess it doesn't hurt